HOWTO/INSTALL.md

```diff
@@ -549,48 +549,54 @@ If you have Xcode 4.3, or later, you will also need to download
 
 #### Building with wxErlang ####
 
-If you want to build the `wx` application, you will need to get wxWidgets-3.0
-(`wxWidgets-3.0.3.tar.bz2` from <https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.3/wxWidgets-3.0.3.tar.bz2>) or get it from github with bug fixes:
+wxWidgets-3.2.x is required for building the `wx` application.
```

Comment:
Replace **required** with **recommended** since it still works to build with wxWidgets-3.0.x